src/NuGet/Microsoft.Net.Compilers.nuspec

```diff
@@ -43,22 +45,34 @@ Supported Platforms:
     <file src="Dlls\MSbuildTask\Microsoft.CSharp.Core.targets" target="tools" />
     <file src="Dlls\MSbuildTask\Microsoft.VisualBasic.Core.targets" target="tools" />
 
-    <file src="Exes\csc\System.Collections.Immutable.dll" target="tools" />
-    <file src="Exes\csc\System.Diagnostics.FileVersionInfo.dll" target="tools" />
-    <file src="Exes\csi\System.Diagnostics.StackTrace.dll" target="tools" />
-    <file src="Exes\csc\System.IO.Compression.dll" target="tools" />
-    <file src="Exes\csc\System.IO.FileSystem.dll" target="tools" />
-    <file src="Exes\csc\System.IO.FileSystem.Primitives.dll" target="tools" />
-    <file src="Exes\csc\System.Reflection.Metadata.dll" target="tools" />
-    <file src="Exes\csc\System.Runtime.InteropServices.RuntimeInformation.dll" target="tools" />
-    <file src="Exes\csc\System.Security.Cryptography.Algorithms.dll" target="tools" />
-    <file src="Exes\csc\System.Security.Cryptography.Encoding.dll" target="tools" />
-    <file src="Exes\csc\System.Security.Cryptography.Primitives.dll" target="tools" />
-    <file src="Exes\csc\System.Security.Cryptography.X509Certificates.dll" target="tools" />
-    <file src="Exes\csc\System.Text.Encoding.CodePages.dll" target="tools" />
-    <file src="Exes\csc\System.Xml.XmlDocument.dll" target="tools" />
-    <file src="Exes\csc\System.Xml.XPath.dll" target="tools" />
-    <file src="Exes\csc\System.Xml.XPath.XDocument.dll" target="tools" />
-    <file src="Exes\csi\System.ValueTuple.dll" target="tools" />
+    <!-- The facades are not signed by us and any deployed copy can be used.  The Deployment
```

Comment:
I'd replace "facades" with "assemblies" since the list contains non-facades CoreFX assemblies as well as facades.